addons/binding/org.openhab.binding.network/ESH-INF/thing/thing-types.xml

```diff
@@ -45,9 +45,17 @@
 				<label>Use System Ping</label>
 				<description>Uses the ping program from the system</description>
 				<default>false</default>
-			</parameter>		
+			</parameter>
+			<parameter name="arping" type="boolean">
+				<label>Use arping</label>
+				<description>Use arping instead of regular ping. Only works on Unix and arping needs to be installed. When using arping it is required to specify a network interface.</description><default>false</default>
```

Comment:
Why is the network interface required? When I run the command from a Linux command line it seems to work fine without the network interface argument. Are you running it on a host with multiple NICs?